pkg/network/tracer/tracer.go

```diff
@@ -862,6 +863,14 @@ func (t *Tracer) DebugNetworkMaps() (*network.Connections, error) {
 	return &network.Connections{Conns: latestConns}, nil
 }
 
+// DebugEBpfMaps returns all maps registred in the eBPF manager
+func (t *Tracer) DebugEBpfMaps() (string, error) {
```

Comment:
casing is strange. I would do `DebugEBPFMaps` to be idiomatic.

Reply:
done